packages/core/exports/vue.ts

```diff
@@ -23,9 +29,29 @@ export function useAppKitAccount() {
     state.value.status = val ?? null
   })
 
+  const unsubscribeAccountDeployed = AccountController.subscribeKey('smartAccountDeployed', val => {
+    state.value.user.isSmartAccountDeployed = Boolean(val)
+  })
+
+  const unsubscribeAccountType = AccountController.subscribeKey('preferredAccountType', val => {
+    state.value.user.accountType = val ?? null
+  })
+
+  const unsubscribeEmail = AccountController.subscribeKey('email', val => {
+    state.value.user.email = val ?? null
+  })
+
+  const unsubscribeUsername = AccountController.subscribeKey('username', val => {
+    state.value.user.username = val ?? null
+  })
```

Comment:
they are all updated at the same time (on connection) so I'd keep a single object for this and we serialize all data as is to the hook. wyt?

Reply:
yeap let me do that